libshumate: unstable-2021-10-06 -> 1.0.0.alpha.1

r-ryantm opened the pull request:
Automatic update generated by [nixpkgs-update](https://github.com/ryantm/nixpkgs-update) tools. This update was made based on information from passthru.updateScript.


meta.description for libshumate is: "GTK toolkit providing widgets for embedded maps"

meta.homepage for libshumate is: "https://gitlab.gnome.org/GNOME/libshumate"

meta.changelog for libshumate is: ""


###### Updates performed
- Ran passthru.UpdateScript

###### To inspect upstream changes


https://gitlab.gnome.org/GNOME/libshumate/-/compare/7a0a03f299881e8faaac7d904cc47b74795ae5dd...1.0.0.alpha.1


###### Impact

<details>
<summary>
<b>Checks done</b> (click to expand)
</summary>

---

- built on NixOS
- The tests defined in `passthru.tests`, if any, passed

- 0 of 0 passed binary check by having a zero exit code.
- 0 of 0 passed binary check by having the new version present in output.
- found unstable-2021-11-12 with grep in /nix/store/dh02sfc2sv39r83h82rxbycp8x4jax6a-libshumate-unstable-2021-11-12
- found unstable-2021-11-12 in filename of file in /nix/store/dh02sfc2sv39r83h82rxbycp8x4jax6a-libshumate-unstable-2021-11-12

---

</details>
<details>
<summary>
<b>Rebuild report</b> (if merged into master) (click to expand)
</summary>

```
2 total rebuild path(s)

2 package rebuild(s)

2 x86_64-linux rebuild(s)
0 x86_64-darwin rebuild(s)


First fifty rebuilds by attrpath
ashpd-demo
libshumate
```

</details>

<details>
<summary>
<b>Instructions to test this update</b> (click to expand)
</summary>

---

Either **download from Cachix**:
```
nix-store -r /nix/store/dh02sfc2sv39r83h82rxbycp8x4jax6a-libshumate-unstable-2021-11-12 \
  --option binary-caches 'https://cache.nixos.org/ https://nix-community.cachix.org/' \
  --option trusted-public-keys '
  nix-community.cachix.org-1:mB9FSh9qf2dCimDSUo8Zy7bkq5CX+/rkCWyvRCYg3Fs=
  cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY=
  '
```
(The Cachix cache is only trusted for this store-path realization.)
For the Cachix download to work, your user must be in the `trusted-users` list or you can use `sudo` since root is effectively trusted.

Or, **build yourself**:
```
nix-build -A libshumate https://github.com/r-ryantm/nixpkgs/archive/cca7ea845a9eb5c9f1e001ec8c5883449354253a.tar.gz
```

After you've downloaded or built it, look at the files and if there are any, run the binaries:
```
ls -la /nix/store/dh02sfc2sv39r83h82rxbycp8x4jax6a-libshumate-unstable-2021-11-12
ls -la /nix/store/dh02sfc2sv39r83h82rxbycp8x4jax6a-libshumate-unstable-2021-11-12/bin
```

---

</details>
<br/>



### Pre-merge build results

We have automatically built all packages that will get rebuilt due to
this change.

This gives evidence on whether the upgrade will break dependent packages.
Note sometimes packages show up as _failed to build_ independent of the
change, simply because they are already broken on the target branch.

Result of `nixpkgs-review` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package failed to build:</summary>
  <ul>
    <li>ashpd-demo</li>
  </ul>
</details>
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>libshumate</li>
  </ul>
</details>

---

###### Maintainer pings

cc @hedning @jtojnar @dasj19 @maxeaubrey for [testing](https://github.com/ryantm/nixpkgs-update/blob/master/doc/nixpkgs-maintainer-faq.md#r-ryantm-opened-a-pr-for-my-package-what-do-i-do).

bobby285271 commented:
@ofborg build ashpd-demo

jtojnar commented:
`append_map_source` method has been removed in https://gitlab.gnome.org/GNOME/libshumate/-/commit/ec6552681e809e06ac4d88ff0df6ee6c9d10b5c1, we need to wait for ashpd-demo to be updated.

r-ryantm committed:
libshumate: unstable-2021-10-06 → 1.0.0.alpha.1

https://gitlab.gnome.org/GNOME/libshumate/-/blob/1.0.0.alpha.1/NEWS

bobby285271 committed:
ashpd-demo: 0.0.1-alpha → 0.2.2

https://github.com/bilelmoussaoui/ashpd/releases/tag/0.2.0
https://github.com/bilelmoussaoui/ashpd/releases/tag/0.2.1
https://github.com/bilelmoussaoui/ashpd/releases/tag/0.2.2

bobby285271 commented:
@ofborg build ashpd-demo libshumate

jtojnar commented on pkgs/development/libraries/libshumate/default.nix:
```diff
@@ -68,11 +68,14 @@ stdenv.mkDerivation rec {
   postFixup = ''
     # Cannot be in postInstall, otherwise _multioutDocs hook in preFixup will move right back.
-    moveToOutput share/doc/libshumate-0.0 "$devdoc"
+    moveToOutput share/doc/libshumate-${lib.versions.majorMinor version} "$devdoc"
```
ABI version is not coupled to package version:

https://gitlab.gnome.org/GNOME/libshumate/-/blob/6404253a6298e1271e564f9fd8e521a2fd30e0f5/meson.build#L27

dasj19 reviewed (approved):
Builds and runs on my system

jtojnar commented:
Thanks.
